app/client/src/components/Panel/Panel.tsx

```diff
@@ -19,14 +19,15 @@ interface PanelProps extends PaperProps {
   responseBody?: object;
   isLoading?: boolean;
   title?: string;
-  apiCallType?: 'GET' | 'POST' | 'DELETE';
+  apiCallType?: 'GET' | 'POST' | 'DELETE' | 'PATCH';
```

Comment:
Any reason the call type is optional?

Reply:
extensibility basically, you don't need to have it to render a panel